src/Features/Core/Portable/PullMemberUp/AbstractPullMemberUpRefactoringProvider.cs

```diff
@@ -65,7 +70,15 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 return;
             }
 
-            PullMemberUpViaQuickAction(context, selectedMember, allDestinations);
+            var allActions = allDestinations.SelectAsArray(
+                destination => MembersPuller.Instance.TryComputeCodeAction(document, selectedMember, destination)).
+                WhereAsArray(action => action != null).
+                Concat(new PullMemberUpWithDialogCodeAction(document, selectedMember, this));
```

Comment:
no need for multiple 'AsArray' calls.  In tihs case, just use normal linq methods (.Select/.WhereNotNull/.Concat) with a final ToImmutableArray.